### PR TITLE
issue-57 Distance filter show meters

### LINKED ISSUE #57: Improve distance filter to show meters

### PR DESCRIPTION
**Pull request by Mxdcq**
Fixes #57

### CODE REVIEW

File: app/js/event/event.filter.js [hunk 1]
```diff
@@ -11,13 +11,19 @@
...
       var distance;
-      if(isNaN(x)){
+      if (isNaN(x)) {
         return x;
-      } else {
+      }
+      else if (x > 1) {
         distance = x.toFixed(2);
-        return distance;
+        return distance + "km";
```
Comment (antti-TIEA4) [inline app/js/event/event.filter.js]: This is not the preferred way to display the units. Could you check out how units are handled e.g. in the built-in [currency](https://docs.angularjs.org/api/ng/filter/currency) filter and do the same in this filter?

**Comment by antti-TIEA4** [comment]
Have you tested that your filter handles the examples below correctly?

| input | | output |
|-----------|-------------------|----------|
| 2.7 km | :arrow_right: | 2.7 km |
| 0.666 km | :arrow_right: | 670 m |
| 0.053 km | :arrow_right: | 55 m |

File: app/js/event/event.filter.js (hunk 1 above)
Comment (Mxdcq) [inline app/js/event/event.filter.js]: Done

File: app/index.html [hunk 2]
```diff
@@ -54,7 +54,8 @@
             <span ng-if="event.venue">@{{ event.venue }}</span>
           </span>
           <p ng-if="event.description" class="description">{{ event.description }}</p>
-          <p ng-if="event.distanceToUser" class="distance description">{{ event.distanceToUser | distanceFormat}} km</p>
+          <p ng-if="event.distanceToUser <= 1" class="distance description">{{ event.distanceToUser | distanceFormat : "m"}}</p>
+          <p ng-if="event.distanceToUser > 1" class="distance description">{{ event.distanceToUser | distanceFormat : "km"}}</p>
```
Comment (Pikkuninja) [inline app/index.html]: Can the distanceFormat be set with ?:-operator (distanceToUser > 1 ? "km" : "m")? That would cut the need for two p elements doing the same thing.

**Comment by Mxdcq** [comment]
I think so.
screen shot 2016-12-14 at 8 22 54 pm

File: app/index.html (hunk 2 above)
Comment (Mxdcq) [reply app/index.html]: I am not sure if it is possible to use Filter in that way.

Comment (Pikkuninja) [reply app/index.html]: I quickly tested it with a filter I have in city info app and it seems to work.

Comment (Mxdcq) [reply app/index.html]: yeah. it works

**Comment by antti-TIEA4** [comment]
Well done! :tada:

**Comment by Mxdcq** [comment]
Haha.

**Pull request merged**